txpipe/twopoint.py

```diff
@@ -1,3 +1,4 @@
+from multiprocessing import Value
```

Comment:
What does this do?

Reply:
Hmm... I have no idea. I'm pretty sure I didn't add this. I'll delete it anyway since 'Value' doesn't seem to be used anywhere.